.github/workflows/update-gradle-cache.yml

```diff
@@ -16,15 +15,12 @@ jobs:
   update-cache:
     name: Update Grade Cache
     runs-on: ubuntu-latest
-
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v4
-
```

Comment:
why remove the whitespace? I thought it made where tasks start/end clearer.

Reply:
The `- name:` is pretty clear delimiters to me but it's just whitespace so I can add them back if you want